examples/list.rkt

```diff
@@ -1,11 +1,32 @@
 #lang racket/base
 
 (module+ test
-  (require rackcheck
+  (require racket/function
+           rackcheck
            rackunit)
 
   (define-property prop-list-reverse
     ([xs (gen:list gen:natural)])
     (check-equal? (reverse (reverse xs)) xs))
 
-  (check-property prop-list-reverse))
+  (define-property prop-list-map-identity
+    ([xs (gen:list gen:natural)])
+    (check-equal? (map identity xs) xs))
+
+  (property-name prop-list-map-identity)
+
+  (define-property prop-list-map-composition
+     ([xs (gen:list gen:natural)])
+     (let ([f (lambda (n) (* 3 (add1 n)))]
+           [g (lambda (n) (/ n 2))])
+      (check-equal? (map f (map g xs)) (map (compose f g) xs))))
```

Comment:
Another nitpick: you should usually prefer `compose1` over `compose` (see the note at the end of `compose1`'s documentation).

Reply:
Done: https://github.com/Bogdanp/rackcheck/pull/9/commits/8dddea7a08d26b0770763ba63e25593946d54806

I was not aware of that, thank you :)